From cbd132f389322ad2239ad57c6cdb9b6bbdc6f4a7 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Mon, 13 Jan 2020 00:20:53 -0800
Subject: [PATCH 0815/1852] dt-bindings: soc: xilinx: Update vcu compatible
 string in example

commit 70acaefc1aecf2fb81315e8f684176751a56fdff from
https://github.com/Xilinx/linux-xlnx.git

Use compatible string with version.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Rohit Visavalia <rohit.visavalia@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt b/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
index 8928ff224465..98474f2accca 100644
--- a/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
+++ b/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
@@ -48,7 +48,7 @@ Optional properties:
 Example:
 
 	xlnx_vcu: vcu@a0040000 {
-		compatible = "xlnx,vcu";
+		compatible = "xlnx,vcu-logicoreip-1.0";
 		#address-cells = <2>;
 		#size-cells = <2>;
 		reg = <0x0 0xa0040000 0x0 0x1000>,
-- 
2.31.1

